data/alephium-official-wallet

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
The whole file should end with the `.json` extension. Yours doesn't have that right now :)